app/pb_kits/playbook/pb_button/docs/_button_default.jsx

```diff
@@ -4,26 +4,22 @@ import { Button } from '../../'
 const ButtonDefault = () => (
   <div>
     <Button
-        dark
-        marginRight="xl"
         onClick={() => alert('button clicked!')}
         text="Button Primary"
     />
+    {' '}
```

Comment:
Is there a better way to add spacers instead of a `{' '}`?

Reply:
Have you tried using class pb--doc-demo-row?

Reply:
I had tried that and it creates them to all be spaced out. I added spacers on react side since rails is displaying the spacing without adding in breaks into the code. If we want to use what you mentioned above across the board for every button example (rails and react) I can add that. But it will no longer left align all the buttons going down the page. Thoughts?

Reply:
<img width="1529" alt="Screen Shot 2020-08-27 at 1 02 17 PM" src="https://user-images.githubusercontent.com/53874143/91473060-59419580-e866-11ea-8cf4-eaba7ca0ae66.png">

Reply:
Could also potentially wrap them in a flex kit as well to create the separation between them.